build.gradle.kts

```diff
@@ -271,44 +272,29 @@ allprojects {
 
         plugins.apply("com.jfrog.bintray")
 
-        bintray {
-            user = "${rootProject.extra["bintray_username"]}"
-            key = "${rootProject.extra["bintray_apiKey"]}"
-            publish = true
-            pkg.apply {
-                userOrg = "hivemq"
-                repo = "HiveMQ"
-                name = "hivemq-mqtt-client"
-                desc = project.description
-                websiteUrl = metadata.url.get()
-                issueTrackerUrl = metadata.issueManagement!!.url.get()
-                vcsUrl = metadata.scm!!.url.get()
-                setLicenses(metadata.license!!.shortName.get())
```

Comment:
is the license handling missing?

Reply:
No, this is done by the metadata plugin.